### PR TITLE
Feature/159 link dev docs to design

### PR DESCRIPTION
**Pull request by MerlindlH**
issue: https://github.wdf.sap.corp/gd/FioriExperience/issues/159

Adds 3 dropdowns to be able to choose the corresponding fundamentals/ngx/react file to a design-control element

Note:  
The current implementation of the FileSelectControl-component is a copy of the netlify-SelectControl component. This way was chosen (e.g. instead of inheritance) to be independent of changes to the netlify-component

TODOs left:
  - [ ] maybe rethink the relation between our new FileSelectControl and the netlify-SelectControl component
  - [ ] find a good way to link angular/react components
  - [ ] code cleanup :)

### CODE REVIEW

File: src/cms/FileSelectControl.js [hunk 1]
```diff
@@ -0,0 +1,215 @@
...
+//             edges {
+//                 node {
+//                     id
+//                     name
+//                     sourceInstanceName
+//                     absolutePath
+//                 }
+//             }
+//         }
+//     }`;
```
Comment (devinea) [inline src/cms/FileSelectControl.js]: remove comments

File: src/cms/FileSelectControl.js [hunk 2]
```diff
@@ -0,0 +1,215 @@
...
+
+  componentDidMount() {
+    //https://github.com/netlify/netlify-identity-widget/issues/144#issuecomment-396868911
+
+    const {field} = this.props;
+    const currentUser = netlifyIdentity.currentUser();
+    if( currentUser && (!this.state || !this.state.ajaxResponse) ){
+      currentUser.jwt().then(accessToken => {
+        const Http = new XMLHttpRequest();
+        //const url= 'https://api.github.com/repositories/162210062/contents/static/docs/fundamental/components';
```
Comment (devinea) [inline src/cms/FileSelectControl.js]: remove

File: src/cms/FileSelectControl.js [hunk 3]
```diff
@@ -0,0 +1,215 @@
...
+//         }
+//     }`
+//   }
+//                render={(data) => (
+//                  <FileSelectControl
+//                    data={data}
+//                    {...props}
+//                  />
+//                )}/>
+// );
```
Comment (devinea) [inline src/cms/FileSelectControl.js]: remove

File: static/admin/test.yaml [hunk 4]
```diff
@@ -0,0 +1 @@
+['a','ab','abc']
```
Comment (devinea) [inline static/admin/test.yaml]: remove

File: static/img/test2.html [hunk 5]
```diff
@@ -5,6 +5,6 @@
     <title>Title</title>
 </head>
 <body>
-
+<h1>lol</h1>
 </body>
 </html>
```
Comment (devinea) [inline static/img/test2.html]: delete file

File: src/cms/FileSelectControl.js (hunk 1 above)
Comment (MerlindlH) [reply src/cms/FileSelectControl.js]: already on it :)

**Comment by MerlindlH** [comment]
@devinea I think there are some things in the new component that could be removed because the are not needed for our purpose (e.g. selecting multiple items), do you think we should remove those?

**Comment by devinea** [comment]
@MerlindlH lets keep multiselect etc.. for now until we lock down the requirements

**Pull request merged**